pkg/admission/interfaces.go

```diff
@@ -36,4 +36,19 @@ type Attributes interface {
 type Interface interface {
 	// Admit makes an admission decision based on the request attributes
 	Admit(a Attributes) (err error)
+
+	// Handles returns true if this admission controller can handle the given operation
+	// where operation can be one of CREATE, UPDATE, DELETE, or CONNECT
+	Handles(operation Operation) bool
 }
+
+// Operation is the type of resource operation being checked for admission control
+type Operation string
+
+// Operation constants
+const (
+	CREATE  Operation = "CREATE"
```

Comment:
Effective Go style conventions, and the standard library all appear to use MixedCase for constants.

please update to just admission.Create, admission.Update, admission.Delete, admission.Connect